vendor/sunnyvalley/+POST_INSTALL

```diff
@@ -0,0 +1,15 @@
+
+if [ ! -f /usr/local/sbin/opnsense-version ]; then
+	echo "OPNsense not installed, please have a proper OPNsense installation first"
+	exit 1
+fi
+
+ISLIBRESSL=$(/usr/local/sbin/opnsense-version | grep LibreSSL)
+
+if [ -z "$ISLIBRESSL" ]; then
+	echo "Configuring repository for OpenSSL flavor"
+	cp -p /usr/local/etc/pkg/repos/SunnyValley.conf.default /usr/local/etc/pkg/repos/SunnyValley.conf
```

Comment:
Coupled with the +POST_DEINSTALL this should work fine, although your effective repo file is not under checksum control in the package. The core repo file isn't either for a number of modifications so that is the same thing here and the right way to do it. :)